pkg/resources/cloudcontroller/openstack.go

```diff
@@ -141,16 +141,16 @@ func getOSFlags(data *resources.TemplateData) []string {
 	return flags
 }
 
-const latestOpenstackCCMVersion = "1.21.1"
+const latestOpenstackCCMVersion = "1.22.0"
```

Comment:
Do we even need the constant anymore? IIRC it was only added because a linter complained.

Reply:
Let's try without it and see if linter is going to complain again.

Reply:
oh i didn't see it in the diff that the constant was still used twice.